pkg/triggers/helper.go

```diff
@@ -2,7 +2,7 @@ package triggers
 
 import (
 	"crypto/hmac"
-	"crypto/sha1"
```

Comment:
Did you verify that test runs successfully?

Reply:
I ran `gauge run --log-level=debug --verbose --tags 'e2e & !tls' specs/triggers` and test completed successfully.